actors/builtin/miner/miner_actor.go

```diff
@@ -280,7 +290,7 @@ func (a Actor) SubmitWindowedPoSt(rt Runtime, params *SubmitWindowedPoStParams)
 		err = st.AddPoStSubmissions(postedPartitions)
 		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to record submissions for partitions %s", params.Partitions)
 
-		// If the PoSt was successful, the declared recoveries should be restored
+		// If the PoSt was successful, the restore declared recoveries.
```

Comment:
nit: did you mean "then restore" or am I just failing to parse the new phrasing?